emsdk.py

```diff
@@ -812,40 +819,58 @@ def git_recent_commits(repo_path, n=20):
     return []
 
 
-def git_clone(url, dstpath, branch):
+def get_git_remotes(repo_path):
+  remotes = []
+  output = subprocess.check_output([GIT(), 'remote', '-v'], stderr=subprocess.STDOUT, text=True, cwd=repo_path)
+  for line in output.splitlines():
+    remotes += [line.split()[0]]
+  return remotes
+
+
+def git_clone(url, dstpath, branch, remote_name='origin'):
   debug_print('git_clone(url=' + url + ', dstpath=' + dstpath + ')')
   if os.path.isdir(os.path.join(dstpath, '.git')):
-    debug_print("Repository '" + url + "' already cloned to directory '" + dstpath + "', skipping.")
-    return True
+    remotes = get_git_remotes(dstpath)
+    if remote_name in remotes:
+      debug_print(f"Repository {url} with remote '{remote_name}' already cloned to directory '{dstpath}', skipping.")
```

Comment:
I believe this script is still supposed to be python2-runnable, so you can't use f-strings.

Worrying that CI didn't pick that up.

Reply:
I don't think Python 2 is a use case anymore. Nothing works in Python 2 mode in Emscripten anymore, for a long time now.

Reply:
Looks like we lost CI support for checking python2 support in #1183..